openapi/paths/API@system@tenant@{id}.yaml

```diff
@@ -35,8 +35,11 @@ put:
   tags:
     - SystemTenant
   summary: Update the current Tenant
+  deprecated: true
   description: |
     Pause or resume the current tenant.
+    
+    Warning: Since Bonita 9.0, Pause or resume tenant is deprecated. Maintenance API should be used instead.
```

Comment:
```suggestion
    Warning: Since Bonita 9.0, system/tenant API resource is deprecated. use system/maintenance instead to enable the maintenance mode which has the same behavior as pausing the tenant.
```